front/src/store/index.js

```diff
@@ -52,6 +53,31 @@ export default new Vuex.Store({
     },
     logout ({ commit }) {
       commit('CLEAR_USER_DATA');
+    },
+    recoveryPasswordSendEmail ({ commit }, payload) {
+      return new Promise((resolve, reject) => {
+        axios.post(`${serverPath}api/ForgotPassword`,
+          { Email: payload.email }).then(resp => {
+          resolve();
+        }).catch(resp => {
+          reject(resp.data);
+        });
+      });
+    },
+    recoveryPasswordSendPassword ({ commit, state }, passwordandtoken) {
```

Comment:
lower camelCase, so it could be passwordAndToken
if object has more properties, this name will be longer?)
we have good standard name - payload